src/components/InputTypes/index.tsx

```diff
@@ -34,7 +39,7 @@ export default function create_input({ question, public_state }: QuestionProp, h
     /* eslint-disable react/react-in-jsx-scope */
     switch (question.type) {
         case QuestionType.TextArea:
-            result = <TextArea handler={handler}/>;
+            result = <TextArea handler={handler} valid={valid} onBlurHandler={blurHandler} focus_ref={focus_ref}/>;
```

Comment:
You use different names for the same thing `onBlurHandler` across the JSX statements in this file. Pick one and stick with it for clarity.